package-lock.json

```diff
@@ -1528,6 +1528,7 @@
         "anymatch": "2.0.0",
         "async-each": "1.0.1",
         "braces": "2.3.1",
+        "fsevents": "1.1.3",
```

Comment:
Any idea where this comes from? Maybe optional, and only used on Mac?

Reply:
Yes, It's optional module only for mac which speedup file watching. It seems tslint want it

Reply:
I can exclude this change from package-lock